src/frontend/src/flows/faq/index.ts

```diff
@@ -0,0 +1,70 @@
+import { html, render } from "lit-html";
+import { compatibilityChart } from "../../components/compatibilityChart";
+
+import { questions } from "./questions";
+
+// re-export for ease of use
+export { questions } from "./questions";
+
+// The FAQ page
+const pageContent = html`
+  <style>
+    /* briefly flash the question when redirected to a particular question */
+    @keyframes example {
+      0% {
+        background-color: transparent;
+      }
+      50% {
+        background-color: var(--grey-100);
+      }
+      100% {
+        background-color: transparent;
+      }
+    }
+
+    a {
+      color: var(--grey-800);
+    }
+
+    ul {
+      list-style-type: none;
+    }
+
+    li {
+      border-radius: 0.5rem;
+      padding: 0.5rem;
```

Comment:
```suggestion
      padding: 0.5rem 0;
```
I don't think the border radius is doing anything here? Also we're already pretty narrow, so let's use the horizontal space we have to put text on the screen.

Reply:
Agreed! But I've just copied this from the about page. I suggest we change them both together separately.